bukuserver/templates/bukuserver/lib.html

```diff
@@ -20,7 +20,14 @@
     .popup :is(#admin-navbar-collapse, .navbar-toggle) {display: none !important}
     .popup .nav-tabs :is(:first-child, :nth-child(2):not(.active)) > * {display: none}
   </style>
-  <script>opener && (opener !== window) && document.body.classList.add('popup')</script>
+  <script>
+    const POPUP = {{ g.popup|default|tojson }} || (opener && (opener !== window)) || '';
```

Comment:
Same thing for pages that support popups.

The `POPUP` value is saved globally, just in case. (…Though I guess it doesn't make much difference considering the existence of the `.popup` CSS class)